src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -73,6 +81,7 @@ internal object AutoMend : Module(
         }
 
         safeListener<TickEvent.ClientTickEvent> {
+            isMending = false
```

Comment:
why are we... why are we setting this to false every client tick?

Reply:
isMending = true
if (validServerSideRotation() && throwDelayTimer.tick(throwDelay.value.toLong())) {
    playerController.processRightClick(player, world, EnumHand.MAIN_HAND)
}
but yeah i think these lines are not needed because i have to revert changes

Reply:
this variable isn't needed if you combine the two tick event handlers